wiki/Guides/BSS_issues/en.md

```diff
@@ -4,35 +4,39 @@ tags:
+When submitting a beatmap, the **[Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission)** (***BSS***) may occasionally prevent you from uploading your beatmap by warning that `This beatmap you're trying to submit isn't yours!`. This is  often caused by an incorrectly set parameter in the [.osu file](/wiki/Client/File_formats/osu_(file_format)) or a bugged submission cache, and can be resolved by following these steps:
 
 1. Go to the beatmap's folder.
-   - You can get there easily by going to `File` then `Open Song Folder` in the [beatmap editor](/wiki/Client/Beatmap_editor).
-2. Exit osu! (This is to make sure that the changes are applied properly.)
+   - You can get there easily by going to `File` and then `Open Song Folder` in the [beatmap editor](/wiki/Client/Beatmap_editor).
+2. Exit osu!. This is to make sure that all the changes are going to be applied properly.
 3. Open the `.osu` difficulty files with any plain text editor (e.g. Notepad).
-4. Enter your username for the `Creator`. (If your name has spaces, then use spaces.)
-5. Set the `BeatmapID` to `0`.
-6. Set the `BeatmapSetID` to `-1`.
-7. Make sure the beatmap folder name does not start with a string of numbers.
-   - E.g. `1000 - Song Name` should be `Song Name`.
-8. Delete the file(s) in the `SubmissionCache` folder. This is located in the osu! installation folder.
+4. Make sure that your username is already written correctly in the `Creator` field. If it isn't, update this field with your current username.
+5. Set the `BeatmapID` field to `0`.
+6. Set the `BeatmapSetID` field to `-1`.
+7. Make sure that the beatmap's folder name does not begin with a string of numbers. If it does, rename this folder accordingly.
+   - For example, `1000 - Song Name` should be renamed into e.g. `ABCDE - Song Name`.
+8. Delete all the files in the `SubmissionCache` folder. This folder can be found in your osu! installation directory.
 9. Open osu! again and try to upload your beatmap.
 
-If you are still having issues, follow these steps:
+You should be able to submit your beatmap by now. If you are still having issues by then, follow these steps:
 
-1. Export the beatmap using the beatmap editor.
-   - `File` then `Export Package`.
-2. Exit the beatmap editor and then delete the beatmap in osu!.
-3. Exit osu!
-4. Go to the exported beatmap and change the extension from `.osz` to `.zip` (you will need file extensions to be visible from the windows folder options to do this).
+1. Export the beatmap from inside the beatmap editor by going to `File` and then `Export Package`.
+2. Exit the beatmap editor and delete the beatmap in osu!.
+3. Exit osu!.
+4. Go to the exported beatmap and change its extension from `.osz` to `.zip`. You will need file extensions to be visible from the Windows folder options in order to do this.
```

Comment:
*Technically* not needed. The file can directly be opened via a user's zip manager of choice (7Z, Winrar, etc...). Not sure if it's worth mentioning, but just as a heads-up that this may be a faster option.